gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryError.java

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
I can try. It seems rude to add a validator to someone else's project, though, so for now I'll only put it the nio side of things.